src/core/render/jsx/factory.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
why disable lint on the whole file? I would rather if we disable specific rules on specific line numbers.